lib/Makefile

```diff
@@ -56,7 +56,8 @@ install: lib-c lib-a lib-py lib/c/criu.pc.in
 	$(Q) install -m 644 lib/c/criu.pc $(DESTDIR)$(LIBDIR)/pkgconfig
 ifeq ($(PYTHON),python3)
 	$(E) "  INSTALL " crit
-	$(Q) $(PYTHON) -m pip install --upgrade --force-reinstall --prefix=$(DESTDIR)$(PREFIX) ./crit
+	$(Q) PIP_BREAK_SYSTEM_PACKAGES=1 $(PYTHON) -m pip install -r ./crit/requirements.txt
```

Comment:
I don't think that this is a good idea to set such variables here.  Let's users to decide whether they wait to "break their systems".

Reply:
If we remove this variable `pip install` / `pip uninstall` fails on Arch linux.
Example: https://github.com/checkpoint-restore/criu/actions/runs/5382664987/jobs/9768409154?pr=2217
```
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try 'pacman -S
    python-xyz', where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Arch-packaged Python package,
    create a virtual environment using 'python -m venv path/to/venv'.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip.
    
    If you wish to install a non-Arch packaged Python application,
    it may be easiest to use 'pipx install xyz', which will manage a
    virtual environment for you. Make sure you have python-pipx
    installed via pacman.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
Error uninstalling package crit: Command '['/usr/sbin/python3', '-m', 'pip', 'uninstall', '-y', 'crit']' returned non-zero exit status 1.
```

Should we skip the install / uninstall of CRIT with externally managed Python environment when `PIP_BREAK_SYSTEM_PACKAGES` is not set?

Reply:
If a user decides to ignore the error, they can set the variable and execute make:
$ PIP_BREAK_SYSTEM_PACKAGES=1 make install

or
$ export PIP_BREAK_SYSTEM_PACKAGES=1
$ make install

Reply:
I agree, we should not set this variable. Maybe if the installation fails you could print a message about the existence of the variable.

Reply:
Thanks Adnrei and Adrian! I have updated the pull request.